Tools/CISupport/ews-build/steps.py

```diff
@@ -493,8 +493,16 @@ def load(cls, use_network=None):
             github_username = value.get('github')
             if name and emails:
                 bugzilla_email = emails[0].lower()  # We're requiring that the first email is the primary bugzilla email
-                cls.contributors[bugzilla_email] = {'name': name, 'status': value.get('status')}
-                cls.contributors[name] = {'status': value.get('status')}
+                cls.contributors[bugzilla_email] = {
+                    'name': name,
+                    'status': value.get('status'),
+                    'email': bugzilla_email,
+                }
+                cls.contributors[name] = {
+                    'name': name,
+                    'status': value.get('status'),
+                    'email': bugzilla_email,
```

Comment:
Nit: not sure if we should call it bugzilla email anymore or just email.

Reply:
Good point, will change